### PR TITLE
Add WNP 126 (Fixes #14512)

### LINKED ISSUE #14512: WNP 126 EU + NA
Hello.

[HERE](https://docs.google.com/document/d/1AKws9Zwr6Y2miKqRu8JVzy56TTpL1ftIOnhS4smPm3Q/edit) is the brief doc for WNP 126. 

Go live is **May 14, 2024.**

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Adds WNP for Firefox 126 for NA and EU locales.

## Issue / Bugzilla link

#14512

## Testing

- http://localhost:8000/en-US/firefox/126.0/whatsnew/
- http://localhost:8000/en-GB/firefox/126.0/whatsnew/
- http://localhost:8000/de/firefox/126.0/whatsnew/
- http://localhost:8000/fr/firefox/126.0/whatsnew/
- http://localhost:8000/es-ES/firefox/126.0/whatsnew/
- http://localhost:8000/it/firefox/126.0/whatsnew/
- http://localhost:8000/pl/firefox/126.0/whatsnew/

### CODE REVIEW

**Comment by alexgibson** [comment]
This is code complete and ready for review, but do-not-merge still until we've had approvals signed off.

**Comment by alexgibson** [comment]
This has been approved, so ready to merge once reviewed.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx126-na.html
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx126-na.html:26]: **Todo:** Needs a closing div.

File: bedrock/firefox/tests/test_base.py
Comment (stephaniehobson) [inline bedrock/firefox/tests/test_base.py:898]: **Nit:** Comment needs updating to en-ES

**Review by stephaniehobson — APPROVED** [review]
2 trivial changes.

R+  ✏️

**Comment by alexgibson** [comment]
Thanks @stephaniehobson, updated 👍

**Pull request merged**
